uefi/src/table/system.rs

```diff
@@ -225,12 +225,13 @@ impl SystemTable<Boot> {
     ///
     /// [`allocator::exit_boot_services`]: crate::allocator::exit_boot_services
     /// [`Logger::disable`]: crate::logger::Logger::disable
-    /// [`uefi_services::init`]: https://docs.rs/uefi-services/latest/uefi_services/fn.init.html
     #[must_use]
     pub fn exit_boot_services(
         self,
         memory_type: MemoryType,
     ) -> (SystemTable<Runtime>, MemoryMap<'static>) {
+        crate::helpers::exit();
```

Comment:
It didn't occur to me this could just be a simple function call now rather than using events, this is much nicer!